### ISSUE TITLE
Possible to have Verbose output by default

### ISSUE THREAD

**Issue by Dedsec1** [mmozeiko/RcloneBrowser#45]
Hey mmozeiko,

not sure if you noticed but in rclone 1.36 the Verbose output seems to be turned off  for some reason, i know rclone browser has the option to enable verbose output before an job starts but is their an way to set it by default in an config file of sorts?

**Comment by mmozeiko** [comment]
Yes, I plan to update UI to show proper progress output by default for 1.36 version.

**Comment by Dedsec1** [comment]
sweet

**Comment by Synopis** [comment]
Correct me if I'm wrong but doesn't a option stay enabled after you use it once?

**Comment by mmozeiko** [comment]
Yes, transfer dialog should remember settings used in previous operation. Does it not happen for you?
